### PR TITLE
adds a breadcrumb

### PR DESCRIPTION
**Pull request by lindseybateman**
to the docs, quickstarts and api pages

### CODE REVIEW

File: docs-bread-crumb/index.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  if (item === section) uri = createSectionUri(base, section);
+
+  return dom.li(
+    {className: 'bread-crumb-item', key: item},
+    dom.a({href: uri}, stripFileExt(item))
+  );
+}
+
+// create href link for base
+// str, str, str, -> str
```
Comment (flenter) [inline docs-bread-crumb/index.js]: This probably should be:
// str -> str

File: docs-bread-crumb/index.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  var base = currentLocation[0];
+  var section = currentLocation[1];
+
+  return currentLocation.map(function (item) {
+    return renderLiElement(item, base, section);
+  });
+}
+
+// render a li element
+// str, str -> obj
```
Comment (flenter) [inline docs-bread-crumb/index.js]: This probably should be:
// str, str, str -> obj

File: docs-bread-crumb/index.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+var react = require('react');
+var sticky = require('react-sticky');
+
+var dom = react.DOM;
+
+/**
+ * Create class.
+ */
+
```
Comment (flenter) [inline docs-bread-crumb/index.js]: Remove empty lines after block comments, or add it after line 47 (whatever the convention is here)

File: docs-bread-crumb/index.js (hunk 1 above)
Comment (lindseybateman) [inline docs-bread-crumb/index.js]: check

**Comment by flenter** [comment]
lgtm

File: docs-bread-crumb/index.js (hunk 2 above)
Comment (lindseybateman) [inline docs-bread-crumb/index.js]: check

File: docs-bread-crumb/index.js (hunk 3 above)
Comment (lindseybateman) [inline docs-bread-crumb/index.js]: check

**Pull request merged**